by-language/java-qa/src/test/java/io/crate/qa/JdbcMetaDataTest.java

```diff
@@ -0,0 +1,1238 @@
+package stock_jdbc;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.RowIdLifetime;
+import java.util.Map;
+
+import org.hamcrest.Matchers;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import io.crate.testing.CrateTestCluster;
+
+@RunWith(JUnit4.class)
+public class JdbcMetaDataTest {
```

Comment:
It's a copy from crate-qa, correct?

Reply:
This is true. See https://github.com/crate/crate-qa/pull/278.